sandcastle/api.py

```diff
@@ -396,8 +395,8 @@ def deploy_pod(self, command: Optional[List] = None):
         if self.is_pod_already_deployed():
             self.delete_pod()
 
-        pod_manifest = self.create_pod_manifest(command=command)
-        self.create_pod(pod_manifest)
+        self.set_pod_manifest(command=command)
+        self.create_pod(self.pod_manifest)
```

Comment:
Is this change required? The commit message doesn't mention it or give any reasoning about it.

I think the previous version was more expressive.

Reply:
I realized I haven't described this change precisely in the commit message after I pushed it and wanted to see CI results first, sorry about that. Though you can see how it's used in the new test case.

I need to trick sandcastle into thinking that the pod is using the default command while decreasing the timeout - so the pod manifest is a class attribute which is mutated in the test case. In this way I managed to reproduce Martin's problem in the new test case. I'm all ears if you have a better idea how to do that.

I think it makes sense to start trimming down ways how sandcastle can be invoked only to the one how we are using it in packit-service. That would enable us to drop tenths of lines of code.

Reply:
I see now. Probably removing the possibility to specify a custom command for `run()` could simplify things. This way only the create pod - exec workflow would be supported. But not in this PR, I guess.

Reply:
agreed: https://github.com/packit/sandcastle/issues/110